pkg/app/app.go

```diff
@@ -228,24 +301,31 @@ func (a *App) Start() {
 		}()
 	}
 
-	a.log.Notice("Ansible Service Broker Started")
-	listeningAddress := "0.0.0.0:1338"
-	if a.args.Insecure {
-		a.log.Notice("Listening on http://%s", listeningAddress)
-		err = http.ListenAndServe(":1338",
-			handler.NewHandler(a.broker, a.log.Logger, a.config.Broker))
-	} else {
-		a.log.Notice("Listening on https://%s", listeningAddress)
-		err = http.ListenAndServeTLS(":1338",
-			a.config.Broker.SSLCert,
-			a.config.Broker.SSLCertKey,
-			handler.NewHandler(a.broker, a.log.Logger, a.config.Broker))
+	genericserver, servererr := apiServer(a.log.Logger, a.config, a.args)
+	if servererr != nil {
+		a.log.Errorf("problem creating apiserver. %v", servererr)
 	}
-	if err != nil {
-		a.log.Error("Failed to start HTTP server")
-		a.log.Error(err.Error())
-		os.Exit(1)
+
+	var clusterURL string
+	if a.config.Broker.ClusterURL != "" {
+		if !strings.HasPrefix("/", a.config.Broker.ClusterURL) {
+			clusterURL = "/" + a.config.Broker.ClusterURL
+		} else {
+			clusterURL = a.config.Broker.ClusterURL
+		}
+	} else {
+		clusterURL = ClusterURLPreFix
 	}
+	daHandler := handler.NewHandler(a.broker, a.log.Logger, a.config.Broker, clusterURL)
+
+	genericserver.Handler.NonGoRestfulMux.HandlePrefix(fmt.Sprintf("%v/", clusterURL), daHandler)
+	a.log.Notice("Listening on https://%s", genericserver.SecureServingInfo.BindAddress)
+
+	a.log.Notice("Starting apiserver")
```

Comment:
Shouldn't we note the Broker is started here? ```a.log.Notice("Ansible Service Broker Started")```

Reply:
I mean it does not actually start until the next line and could fail. I would prefer to make it `a.log.Notice("Ansible Service Broker Starting")`? is that agreeable?